src/com/projectkorra/projectkorra/earthbending/EarthSmash.java

```diff
@@ -398,13 +398,14 @@ public void animateLift() {
 				 */
 				this.location.add(0, -1, 0);
 
+				// Move any entities that are above the rock.
+				final List<Entity> entities = GeneralMethods.getEntitiesAroundPoint(this.location, 3.5);
+				for (final Entity entity : entities) {
+					final org.bukkit.util.Vector velocity = entity.getVelocity();
+					entity.setVelocity(velocity.add(new Vector(0, 1.1, 0)));

```

Comment:
I tested it, and it works really well. One concern I have is that it shoots the player up less than it used to when you jumped. I think there are going to be quite a few earthbenders who will want it to be configurable, and/or for the default value to be higher than this.

Reply:
Earthbenders are going to want this to be configurable, and/or having the default value higher than this

Reply:
I would prefer to leave the value as is but I am not opposed to making it configurable.